core/columns_api.php

```diff
@@ -179,9 +179,15 @@ function columns_get_plugin_columns() {
 		foreach( $t_all_plugin_columns as $t_plugin => $t_plugin_columns ) {
 			foreach( $t_plugin_columns as $t_callback => $t_plugin_column_array ) {
 				if( is_array( $t_plugin_column_array ) ) {
-					foreach( $t_plugin_column_array as $t_column_class ) {
-						if( class_exists( $t_column_class ) && is_subclass_of( $t_column_class, 'MantisColumn' ) ) {
-							$t_column_object = new $t_column_class();
+					foreach( $t_plugin_column_array as $t_column_item ) {
+						$t_column_object = null;
+						if( is_object($t_column_item) && $t_column_item instanceof MantisColumn ) {
+							$t_column_object = $t_column_item;
+						}
+						else if( class_exists( $t_column_item ) && is_subclass_of( $t_column_item, 'MantisColumn' ) ) {
+							$t_column_object = new $t_column_item();
+						}
```

Comment:
I would remove `$t_column_object = null;` (line 183) and add 
```
else {
    continue;
}
```
here, then we can get rid of the if() below.